tests/unit/amo/pages/TestGuides.js

```diff
@@ -210,14 +219,60 @@ describe(__filename, () => {
         }),
       );
 
-      _loadAddonResults(store, addon);
+      _loadAddonResults({ store, addons });
 
-      const root = render({ store, slug });
+      const root = render({ _log, store, slug });
+
+      addons.forEach((addon, index) => {
+        expect(root.find(GuidesAddonCard).at(index)).toHaveProp(
+          'addon',
+          createInternalAddon(addon),
+        );
+      });
+
+      sinon.assert.notCalled(_log.error);
+    });
+
+    it('passes addon value as null to GuidesAddonCard with a log message if the guid is not valid', () => {
```

Comment:
passes addon value as `null` to GuidesAddonCard and logs an error when the GUID is not valid